gnucash/python/init.py

```diff
@@ -1,15 +1,16 @@
 import sys
 import gnucash._sw_app_utils as _sw_app_utils
+import gnucash._sw_core_utils as _sw_core_utils
```

Comment:
I think I'd like to make sw_core_utils accessible in the python-bindings as well. That way this path to the locale files could be used there as well.